lib/Parser/Parse.cpp

```diff
@@ -11130,6 +11156,9 @@ HRESULT Parser::ParseFunctionInBackground(ParseNodePtr pnodeFnc, ParseContext *p
     pnodeFnc->sxFnc.pnodeBody = nullptr;
     pnodeFnc->sxFnc.nestedCount = 0;
 
+    // getting the parent node this way might be incorrect
+    // todo check with Paul how to approach this
+    ParseNodePtr pnodeParentFnc = m_currentNodeFunc;
```

Comment:
@pleath is there a way to get the parent function node in parallel parse?  Is parallel parse going to be maintained into the future? #1478 suggests it is not used and could be excluded from build.